catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -210,6 +210,13 @@
             type="Boolean"
             default="true"/>
 
+        <AD id="clientValidationEnabled"
+            name="Enable Validation"
+            description="Enable client-side validation of metacard attributes"
```

Comment:
✏️ This could also be more descriptive. I think we should mention what the validation looks like and which view is impacted, something like `Enable client-side validation of metacard attributes in the Inspector view. Invalid attribute values are marked with a red banner and a message is included to indicate why the value is invalid.`

Reply:
this red banner can technically show up anywhere an input enum view is used. so it could also show up on search forms, search settings, etc. maybe 
```
Enable client-side validation of metacard attributes. Invalid attribute values are marked with a red banner and a message is included to indicate why the value is invalid.
```

Reply:
actually, since they always just say "Not an acceptable Value", maybe something like:

`Enable client-side validation of metacard attributes. Invalid attribute values are marked with a red "Not an acceptable value" banner.
`